test/gtest/uct/ib/test_rc.cc

```diff
@@ -369,3 +369,53 @@ UCS_TEST_P(test_rc_flow_control_stats, soft_request)
 UCT_INSTANTIATE_RC_TEST_CASE(test_rc_flow_control_stats)
 
 #endif
+
+#if HAVE_MLX5_HW
+
+extern "C" {
+    #include <uct/ib/rc/accel/rc_mlx5_common.h>
```

Comment:
should it be aligned as below?
```
extern "C" {
#include <uct/ib/rc/accel/rc_mlx5_common.h>
}
```

Reply:
ok, will fix together with other comments